src/App.css

```diff
@@ -35,4 +36,16 @@
   to {
     transform: rotate(360deg);
   }
+} */
+
+/* Used as the background-color of the appBar */
+#header {
+  background-color: orange;
 }
+
+/* Shrinks the Revature logo a little bit to fit inside the appBar*/
+#image {
+  width:150px;
+  height:auto;
+  margin-top:6px;
+}
```

Comment:
Remember that there are specific margins based on the size of the "R" of revature

Reply:
is it these? I did not see any information regarding specific margin sizes.
body {
font-family: 'futura-pt';
font-size: ‘14px’;
color: #474c55;}